src/Destructurama.FSharp.Tests/Destructurama.FSharp.Tests.fsproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <GenerateProgramFile>false</GenerateProgramFile>
```

Comment:
no need for this or program.fs
(same as C# projects)

Reply:
Thanks. Adressed in https://github.com/destructurama/fsharp/commit/cd5adf371f201c6de509bd680c1efdbf99bf0e81